docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4062,6 +4109,26 @@ supported as follows:
 * If the `index` field is not set, the server returns the configuration for all
   indirect meters in the array identified by the unique id `meter_id`.
 
+### `MeterCounterData`
+
+The `MeterCounterData` P4Runtime message represents the per color counters
+associated with the meter configuration. Whenever a meter is updated and returns
+a color, the corresponding color counter GREEN, YELLOW or RED is updated.
+
+As seen above, these counters can be associated with a `DirectMeterEntry` or
+`MeterEntry`. However, they are implementation specific and targets not capable
+of supporting these counters should return `UNIMPLEMENTED` if the field(s) was
```

Comment:
> if the field(s) was set

Can we just say "if a `MeterCounterData` field is set`? This is more precise and unambiguous.